tests/wpt/mozilla/tests/css/stylesheet_media_queries.html

```diff
@@ -0,0 +1,22 @@
+<!doctype html>
+<meta charset="utf-8">
+<title></title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<iframe id="myframe" src="iframe_for_media_queries.html" height="500px" width="500px" onLoad="func();">
```

Comment:
Func may not exist if the iframe loads too fast. It's unlikely, but please move the definition of `func` above.